doc/man1/flux-update.rst

```diff
@@ -75,7 +75,9 @@ SPECIAL KEYS
   Updates of the job ``duration`` can take the form of of *[+-]FSD*, where
   ``+`` or ``-`` indicate an adjustment of the existing duration, and *FSD*
```

Comment:
The doc commit title has a typo: s/flux-upate(1)/flux-update(1)/

Reply:
D'oh! Fixed and pushed. Will set MWP Thanks!